tests/acceptance/features/bootstrap/WebUISharingContext.php

```diff
@@ -896,6 +896,22 @@ public function theUsersOwnNameShouldNotBeListedInTheAutocompleteList() {
 		);
 	}
 
+	/**
+	 * @Then the user :username should not be listed in the autocomplete list on the webUI
```

Comment:
We don't usually use `the user` when user name is given.
```suggestion
	 * @Then user :username should not be listed in the autocomplete list on the webUI
```

Reply:
https://doc.owncloud.com/server/developer_manual/core/acceptance-tests.html#referring-to-named-entities
and the doc on that page has the "standards" that we try to use for consistency.